jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java

```diff
@@ -71,21 +72,17 @@
           "templating parameter to inject into build file, replace $${<name>} with <value> (repeatable)")
   private Map<String, String> templateParameters = Collections.emptyMap();
 
-  public Path getContextRoot() {
-    return Verify.verifyNotNull(contextRoot);
-  }
-
   /**
    * Returns a user configured Path to a buildfile and if none is configured returns jib.yaml in
-   * {@link #getContextRoot()}.
+   * {@link #contextRoot}.
    *
-   * @return a path to a bulidfile
+   * @return a path to a buildfile
    */
   public Path getBuildFile() {
```

Comment:
Doesn't seem like it needs to be `public`. `@VisibleForTesting`?

Reply:
oh right.